lib/spack/spack/cmd/flake8.py

```diff
@@ -246,7 +245,7 @@ def prefix_relative(path):
 
         with working_dir(spack.paths.prefix):
             if not file_list:
-                file_list = changed_files(args)
+                file_list = changed_files(args.base, args.untracked, args.all)
```

Comment:
Why are there changes to `spack flake8` in this PR? I don't see why these are related.

Reply:
Replied below